server/sonar-server/src/test/java/org/sonar/server/user/ws/CurrentActionTest.java

```diff
@@ -185,7 +195,8 @@ public void json_example() {
       .setExternalIdentity("obiwan.kenobi")
       .setExternalIdentityProvider("sonarqube")
       .setScmAccounts(newArrayList("obiwan:github", "obiwan:bitbucket"))
-      .setOnboarded(true));
+      .setOnboarded(true)
+      .setHomepageType("MY_PROJECTS"));
```

Comment:
why not set the value?

Reply:
because generated uuid and of little value here